cmd/register/register_util.go

```diff
@@ -127,27 +128,37 @@ func hydrateNode(node *core.Node) error {
 			launchPlanNodeReference := workflowNodeWrapper.WorkflowNode.Reference.(*core.WorkflowNode_LaunchplanRef)
 			hydrateIdentifier(launchPlanNodeReference.LaunchplanRef)
 		default:
-			errors.New(fmt.Sprintf("Unknown type %T", workflowNodeWrapper.WorkflowNode.Reference))
+			fmt.Printf("Unknown type %T", workflowNodeWrapper.WorkflowNode.Reference)
 		}
 	case *core.Node_BranchNode:
 		branchNodeWrapper := targetNode.(*core.Node_BranchNode)
-		hydrateNode(branchNodeWrapper.BranchNode.IfElse.Case.ThenNode)
+		if err := hydrateNode(branchNodeWrapper.BranchNode.IfElse.Case.ThenNode); err != nil {
+			// TODO Write Error Log
+			fmt.Println("failed to hydrateNode")
```

Comment:
I think we should return this error - not just log it - @pmahindrakar-oss, we need to fix this

Reply:
Returning an error in earlier code would cause panic in main.go
Now that this is being changed to been logged to stdout
in this change https://github.com/flyteorg/flytectl/pull/17/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R10
then we should be returning this error all the way up. 

Will check and fix after the above mr gets merged.